process/process.go

```diff
@@ -0,0 +1,33 @@
+package process
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+// IncludeFile replaces intsances of <<include(file)>> with the contents
+// of "file", escaping instances of "<<" within the file before returning.
+func IncludeFile(template string, root string) (string, error) {
+	// View: https://regexr.com/582gb
+	includeRegex, err := regexp.Compile(`(?U)^<<\s*include\((.*\/*[^\/]+)\)\s*?>>$`)
```

Comment:
Is the intent here that the entire YAML node must be exactly one `include` stanza? That seems reasonable for scripts, but maybe there are cases where someone will want to do something different. If we do go with that, we should handle user error gracefully.

Cases I'd think about:

- `sh -c '<< include(script) >>'`
- gracefully handling multiple includes in one string?

Currently the first one will be silently ignored and the second one will get an error message like

```
Error: An unexpected error occurred: could not open /home/jds/src/slack-orb/src/scripts/notify.sh) >> << include(scripts/notify.sh for inclusion
```

Reply:
(Sorry I didn't notice this before, since it isn't actually a new part of this PR)

Reply:
This is as intended, there has been a fair amount of dicsussion around this and the conclusion is this is how it is intended to function. Although the error is something we probably want to eliminate.

Reply:
Feel free to merge as is but I definitely agree that we should fix the errors.